backend/tickit/serializers.py

```diff
@@ -0,0 +1,22 @@
+from tickit.models import Item, ItemList
+from rest_framework import serializers
+
+class ItemListSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(max_length=30, required=True)
+
+    class Meta:
+        model = ItemList
+        fields = ['id', 'name']

```

Comment:
For simple ModelSerializers, I think it's a good idea to let DRF handle the fields automatically:

```suggestion
class ItemListSerializer(serializers.ModelSerializer):
    class Meta:
        model = ItemList
        fields = ['id', 'name']
```
Why do I say this?

Well, for example - here we're redefining the maximum length of the `name` field on the serializer. So if we change the number of chars in the database model, we will also need to remember to change this on the serializer as well.

DRF is able to come up with some really sensible defaults based on the `model` attribute, so for most fields we can just let it handle things by itself. What you've done here is a great way to understand what it's doing behind the scenes